src/mc2acab/MCNP_ACAB_library.py

```diff
@@ -515,3 +515,63 @@ def summary_table_gen(totaldata_ACAB,tally,**kwargs):
             totals.to_csv(f'summary_ACAB_{tally.cells[i]}.csv',sep='\t',encoding='utf-8')
     # apypas = np.load('summary_apypas.npy', allow_pickle=True)
     return apypas
+
+def apypa2sdef(n_cell=0, decay_time=0,  infile='summary_apypas.npy'):
+    """ Genera una entrada SDEF a partir de un summary_apypa"""
+    while not os.path.exists(infile):
+        infile = input('summary_apypas.npy not present, please type apypa input file: ')
+    apypas_in = np.load(infile, allow_pickle=True)
+    cells = [x.astype(int) for x in apypas_in['cell']]
+    while int(n_cell) not in cells:
+        print(apypas_in['cell'])
+        n_cell = int(input('Cell number not included in apypa, please type correct one: '))
```

Comment:
What happens if you just want to cancel? Do you force quit?

Reply:
Well... it prompts the correct cells... write at least one of them and it will resume execution